spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/BigQueryStrategy.scala

```diff
@@ -48,18 +49,35 @@ class BigQueryStrategy(expressionConverter: SparkExpressionConverter, expression
    *         query generation was successful, None if not.
    */
   override def apply(plan: LogicalPlan): Seq[SparkPlan] = {
+    // Check if we have any unsupported nodes in the plan. If we do, we return
+    // Nil and let Spark try other strategies
+    if(hasUnsupportedNodes(plan)) {
```

Comment:
Is `order by` is going to be solved by another PR?

Reply:
yes. this wont solve the `order by` problem. the solution for that will have to be investigated